### DISCUSSION TITLE
mhi2_k3663_aio - MMI Android Auto Symbol size

### DISCUSSION THREAD

**Discussion by Lenskoo** [Mr-MIBonk/M.I.B._More-Incredible-Bash#91] [Q&A]
Hello, I'm brand new here and would first like to say thank you for providing updates and upgrades from older systems.
Yesterday I installed the MIB and updated it from mhi2_ER_Au57x_P3236 to K3663. I noticed that Android Auto's icons are much larger than the previous version and take up a lot of the screen. Is there a solution for this and can this be adjusted?

**Comment by harman-f** [comment]
Nothing M.I.B has a solution for.

There was something about changing DPI/resolution in one of the setting files in the unit. 
Can not remember where I read it ;)

This had some impact on the representation of buttons and also could enable split screen mode on AA.

If you find those post again I can try to get you something to run with M.I.B.

**Reply by harman-f** [reply]
https://github.com/jilleb/mib2-toolbox might have a function for that
